dll/win32/shell32/wine/control.c

```diff
@@ -736,6 +736,53 @@ static	void	Control_DoWindow(CPanel* panel, HWND hWnd, HINSTANCE hInst)
 }
 #endif
 
+/** Structure for in and out data when
+ *  search for the cpl dialog of first instance
+ */
+typedef struct tagAppDlgFindData
+{
+    const WCHAR*    szCaption;  /**< second search value */
+    HWND            hResult;    /**< variable for result value */
+} AppDlgFindData;
+
+/**
+ * Callback function to search applet dialog
+ * @param hwnd A handle to a top-level window.
+ * @param lParam Pointer of AppDlgFindData
+ * @return TRUE: continue enumeration, FALSE: stop enumeration
+ */
+static BOOL CALLBACK Control_EnumWinProc(
+  _In_ HWND   hwnd,
+  _In_ LPARAM lParam
+)
+{
+    WCHAR szClassName[256];
+    if (GetClassNameW(hwnd, szClassName,
+                      sizeof(szClassName) / sizeof(WCHAR)))
+    {
+        // check of first search value
+        if (0 == wcscmp(L"#32770", szClassName))
```

Comment:
This should be used to detect typos.
Example: desired expression (var == 0)
  a) Typing error (var = 0) has the consequence that var is assigned with new value and the compiler accepts it.
  b) Typing error (0 = var) has the consequence that the compiler complains.

Reply:
we do know what Yoda style is used for.
I guess hbelusca criticized using that style, because modern static code analysis is also capable to detect bugs of that type.
And non-Yoda is easier to read and easier to understand for the human brain.
Personally I am neutral to that style.

Reply:
> I guess hbelusca criticized using that style

We have a code style to follow for that: https://reactos.org/wiki/Coding_Style#Control_structures